src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const STATUS = 'STATUS';
+
+export const bookStatus = (book) => ({
```

Comment:
- [ ] Please follow the ducks pattern in naming your action types. Kindly refer to the suggestion given above 👆